Database/DatabaseSender.py

```diff
@@ -140,7 +143,8 @@ def insert_antireqs(self, code, antireqs):
         not_exist += "WHERE course_code = '" + code + "'"
         command = "INSERT INTO " + self.antireqs_table + " (course_code, antireq, extra_info)"
         command += "\nSELECT '" + code + "', '" + antireqs.str("antireqs") + "', '" + antireqs.str("extra") + "'\n"
-        command += "WHERE NOT EXISTS (\n" + not_exist + "\n);"
+        command += "FROM dual\n"
```

Comment:
what is dual?

Reply:
WHERE clause needs FROM clause. Dual is just a dummy table for this purpose

Reply:
would be nice if you can add a comment above the code to explain this - so it is more intuitive to other people reading the code